skaffold-plugins-core/src/test/java/com/google/cloud/tools/skaffold/yaml/SkaffoldYamlGeneratorTest.java

```diff
@@ -26,18 +31,14 @@
 public class SkaffoldYamlGeneratorTest {
 
   @Test
-  public void testGenerate() {
+  public void testGenerate() throws URISyntaxException, IOException {
+    String expected =
+        new String(
+            Files.readAllBytes(Paths.get(Resources.getResource("yaml/test-skaffold.yaml").toURI())),
```

Comment:
Could you tie the test resource name to the test?  `SkaffoldYamlGeneratorTest/generated.yaml`?